src/utils/checkButton.ts

```diff
@@ -0,0 +1,23 @@
+export const checkButton = (type: string) => {
+  switch (type) {
+    case 'green-btn':
```

Comment:
Add constants for cases

For example

`const GREEN_STYLE_BUTTON = "green-btn";

case GREEN_STYLE_BUTTON:
`